server/services/qBittorrent/clientRequestManager.ts

```diff
@@ -483,7 +483,7 @@ class ClientRequestManager {
         new URLSearchParams({
           hashes: hashes.join('|').toLowerCase(),
           tags: tags?.join(','),
-        }),
+        } as Record<string, string>),
```

Comment:
this and changes in `server/services/rTorrent/util/scgiUtil.ts` are caused by `@types/node`